src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -497,6 +497,10 @@ export const TradeInput = ({ isCompact }: TradeInputProps) => {
         ]
     }, [activeQuote, buyAssetFeeAsset])
 
+  const isSmartContractBridge = useMemo(() => {
```

Comment:
rationale: the current name doesn't say much, and it seems close to impossible to come up with something descriptive enough, i.e:
- which side is the sc address, the sell or buy side?
- what do we mean by bridge, is it an actual bridge or a bridge + swap, or swap + bridge?

```suggestion
  const shouldForceManualAddressEntry = useMemo(() => {
```